staker/staker.go

```diff
@@ -235,12 +243,13 @@ func NewStaker(
 	}
 	stakerLastSuccessfulActionGauge.Update(time.Now().Unix())
 	if config.StartFromStaked {
-		notifiers = append(notifiers, blockValidator)
+		stakedNotifiers = append(stakedNotifiers, blockValidator)
```

Comment:
the condition should also check if blockValidator isn't nil (my bug from an earlier PR.. but should be fixed here)

Reply:
Thanks, nice catch